src/client/scripts/game/gui/guigameinfo.js

```diff
@@ -32,8 +32,10 @@ const guigameinfo = (function(){
         const white = gameOptions.metadata.White;
         const black = gameOptions.metadata.Black;
         // If you are a guest, then we want your name to be "(You)" instead of "(Guest)"
-        element_playerWhite.textContent = onlinegame.areWeColor('white') && white === translations["guest_indicator"] ? translations["you_indicator"] : white;
-        element_playerBlack.textContent = onlinegame.areWeColor('black') && black === translations["guest_indicator"] ? translations["you_indicator"] : black;
+        element_playerWhite.textContent = (enginegame.areWeColor('white') || onlinegame.areWeColor('white'))
+                                          && white === translations["guest_indicator"] ? translations["you_indicator"] : white;
+        element_playerBlack.textContent = (enginegame.areWeColor('black') || onlinegame.areWeColor('black'))
```

Comment:
Same issue as above (repeat logic). Lets also create a function in game.js called `areWeColor()` that does all this logic for us, no matter what kind of game we're in.